assets/ui/components/ArticleBodyHtml.jsx

```diff
@@ -53,9 +50,9 @@ class ArticleBodyHtml extends React.PureComponent {
         this.players.forEach(player => player.destroy());
         this.players.clear();
 
-        if (this.memoryInterval) {
-            clearInterval(this.memoryInterval);
-        }
+        // if (this.memoryInterval) {
```

Comment:
Please remove commented out code no longer in use, and remove code that is not used anywhere such as `startMemoryUsageTracking`

Reply:
update all